vumi_message_store/tests/test_memory_backend_manager.py

```diff
@@ -0,0 +1,34 @@
+"""
+Tests for vumi_message_store.memory_backend_manager.
+"""
+from vumi.tests.helpers import VumiTestCase
+from vumi_message_store.memory_backend_manager import (
+    FakeRiakState, FakeRiakObject)
+
+
+class TestFakeRiakState(VumiTestCase):
+
+    def setUp(self):
+        self.state = FakeRiakState(is_sync=False)
+        self.add_cleanup(self.state.teardown)
+
+    def test_old_objects_removed_from_indexes(self):
+        bucket = self.state._get_bucket("bucket")
```

Comment:
Test methods need docstrings explaining what they test. See the existing tests in this project for examples.